client/ayon_maya/plugins/publish/collect_render.py

```diff
@@ -118,7 +118,7 @@ def process(self, instance):
         for product in render_products:
             self.log.debug(f"Getting render product: {product}")
             if product.multipart:
-                multipart = True
+                multipart = product.multipart
```

Comment:
Does this really change anything, since it's still in the `if` check, or?

Reply:
It is supposed that renderProduct has stored multipart as boolean value. But to ensure there is any nonetype-value bug, I would use the boolean value as if before.